apps/src/templates/progress/StageExtrasProgressBubble.jsx

```diff
@@ -2,29 +2,34 @@ import React, {Component} from 'react';
 import PropTypes from 'prop-types';
 import Radium from 'radium';
 import _ from 'lodash';
-import assetUrl from '@cdo/apps/code-studio/assetUrl';
 import i18n from '@cdo/locale';
 import TooltipWithIcon from './TooltipWithIcon';
 import {currentLocation} from '../../utils';
+import color from '@cdo/apps/util/color';
 
 const styles = {
   main: {
-    backgroundImage: `url('${assetUrl(
-      'media/common_images/flag_inactive.png'
-    )}')`,
     width: 21,
     height: 24
   },
+  flagNormal: {
+    color: color.white
+  },
+  checkFlagNormal: {
+    color: color.lighter_gray
+  },
   focused: {
-    backgroundImage: `url('${assetUrl('media/common_images/flag_active.png')}')`
+    color: '#32CD32'
```

Comment:
which color is this? i'd like for it to be a constant if possible, so i'm wondering if we should:
1. add this color to our color constants, or
2. use the most similar color that is already in our color constants

Reply:
I'll add it - because every other green color was not even close